inc/template.php

```diff
@@ -1004,15 +1004,11 @@ function tpl_pagetitle($id = null, $ret = false)
         case 'profile':
         case 'register':
         case 'resendpwd':
+        case 'index':
             $page_title = $lang['btn_' . $ACT];
             break;
-
         // wiki functions
         case 'search':
-        case 'index':
-            $page_title = $lang['btn_' . $ACT];
-            break;
-
```

Comment:
Do I interpret it correctly that search does not retrieve the btn_* language string anymore?

Reply:
hmm, you're right. I was focusing on what it did with the index action and missed that it broke the search action. I pushed a fix.